.pre-commit-config.yaml

```diff
@@ -26,7 +26,7 @@ repos:
     rev: 'v0.812'
     hooks:
       - id: mypy
-        args: [ --ignore-missing-imports --exclude '/setup\.py$' ]
```

Comment:
@DidierRLopes I couldn't push my changes with this flag. I replaced it to previous one. I don't know what is it about, but let me know if I need to change it.

That was output when I tried to commit.
```
mypy: can't read file '--ignore-missing-imports --exclude '\setup\.py$'': No such file or directory
```

Reply:
@JakubPluta this should be on the jupyterlab. Can you tell me what's the file giving the issue?

I think you should be able to use this:
```pyupgrade --py36-plus **/*.py```

And that will solve it.

Reply:
It worked I think, I could commit :D Thanks!

Reply:
Nice! So you didn't need to add the `--exclude` line right?

Reply:
I mean to main version was: `[ --ignore-missing-imports --exclude '/setup\.py$' ]` , I couldn't commit my changes on feature branch because mypy screamed `mypy: can't read file '--ignore-missing-imports --exclude '\setup\.py$'': No such file or directory`, so to obey that I removed `--exclude '/setup\.py$'`. 

Now I run your command `pyupgrade --py36-plus **/*.py` and I could change back this line of code to previous version (this which was on main :`args: [ --ignore-missing-imports --exclude '/setup\.py$' ]`)

Sounds complicated, but in the end this line in .pre-commit-config is in initial form nothing changed there.

Reply:
I understand now! Not sure why that exclude is there, but happy to leave it like that since it was already like this.

Let's wait for James to test this, and unless he sees something extremely off we can merge 😄